std/container/rbtree.d

```diff
@@ -2064,3 +2064,13 @@ if ( is(typeof(binaryFun!less((ElementType!Stuff).init, (ElementType!Stuff).init
     static assert(!__traits(compiles, *t3.front.p = 4));
     assert(*t3.front.p == 1);
 }
+
+// make sure the comparator can be a delegate
+@safe pure unittest
+{
+    import std.algorithm.comparison : equal;
+
+    auto t = new RedBlackTree!(int, delegate (a, b) => a > b);
```

Comment:
It's always a good idea to use the more common style:

```
~/dlang/phobos on master [A11] ± ag "delegate\(" |wc
    384    2599   31451

~/dlang/phobos on master [A11] ± ag "delegate \(" |wc
      9      91     795
```

Reply:
Thanks!